src/Physics/HadronTransport/INukeHadroFates2018.h

```diff
@@ -116,6 +116,15 @@ class INukeHadroFates {
      return "undefined"; 
   }
   //__________________________________________________________________________
+
+  static std::map<INukeFateHN_t,string> GetEINukeFateKeysMap( void ) {
```

Comment:
Because this map is inconsistent with the rest, we should move it to the CascateReweight header.

Reply:
since the map itself is static, you can probably return a reference in this case.